gateway-js/src/index.ts

```diff
@@ -90,9 +91,7 @@ type WarnedStates = {
   remoteWithLocalConfig?: boolean;
 };
 
-export const GCS_RETRY_COUNT = 5;
-
-export function getDefaultGcsFetcher() {
```

Comment:
Out-of-band comment from @glasser: this is public API so we should keep it and deprecate it.

Reply:
Resolved in 307ee0eb